yt/visualization/volume_rendering/tests/test_off_axis_SPH.py

```diff
@@ -18,7 +18,7 @@ def test_no_rotation():
     image buffer if processed directly through 
     pixelize_sph_kernel_projection
     """
-    normal_vector = [0., 0., 1.]
+    normal_vector = np.array([0., 0., 1.])
```

Comment:
This should be handled inside of the `yt.off_axis_projection`, I think. It just needs a line at the top that does `normal_vector = np.array(normal_vector, dtype='float64')` and then users will still be able to pass in lists.

Reply:
Oh, good call.  That makes it cleaner.